### DISCUSSION TITLE
Example (d) oscillatory background flow, spheres disappeared instead of getting back to original positions

### DISCUSSION THREAD

**Discussion by ZilH** [Pecnut/stokesian-dynamics#6] [Q&A]
Thank you so much for making this stokesian dynamics code available! 

I was able to run the code but i noticed for the example (e). First, i run it with the parameters `python run_simulation.py 4 3 1 100 fte`

when plotting out the output video, the particles and dumbbells moved out of frames and did not get back to their original positions. At first i thought it could be the time was not enough. So i tried to increase the time step to 1000 but again, those particles just run out with no reverse movements. 

I thought due to the reversibility at low Re number, the particles should go back to their original position? Did you have any idea on this?

**Comment by Pecnut** [comment]
Hi @ZilH, sorry for the slow reply. I've added more information now about how to adjust the viewing frame so that you can see what's going on all the time.

I've added a video for what I see when I run example (d). It looks periodic to me?
